usaspending_api/disaster/v2/views/spending_by_geography.py

```diff
@@ -0,0 +1,203 @@
+import copy
+import json
+from decimal import Decimal
+from enum import Enum
+from typing import Optional, List, Dict
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+from elasticsearch_dsl import A, Q as ES_Q
+
+from usaspending_api.common.cache_decorator import cache_response
+from usaspending_api.common.elasticsearch.search_wrappers import AwardSearch
+from usaspending_api.common.exceptions import UnprocessableEntityException
+from usaspending_api.common.helpers.generic_helper import get_generic_filters_message
+from usaspending_api.common.query_with_filters import QueryWithFilters
+from usaspending_api.common.validator import TinyShield, PAGINATION
+from usaspending_api.common.validator.award_filter import AWARD_FILTER
+from usaspending_api.disaster.v2.views.disaster_base import DisasterBase
+from usaspending_api.search.v2.elasticsearch_helper import get_number_of_unique_terms, get_scaled_sum_aggregations
+
+
+class GeoLayer(Enum):
+    COUNTY = "county"
+    DISTRICT = "district"
+    STATE = "state"
+
+
+class SpendingByGeographyViewSet(DisasterBase):
+    """Spending by Recipient Location"""
+
+    endpoint_doc = "usaspending_api/api_contracts/contracts/v2/disaster/spending_by_geography.md"
+    # TODO: Are contracts under https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts/contracts/v2/disaster/recipient/location needed anymore??
```

Comment:
Those API Contracts are for the endpoints to power the table-view behind the map (post-MVP)